src/Stryker.Core/Stryker.Core/ProjectComponents/TestProjects/TestProjectsInfo.cs

```diff
@@ -16,7 +16,9 @@ public class TestProjectsInfo
     private readonly ILogger<TestProjectsInfo> _logger;
 
     public IEnumerable<TestProject> TestProjects { get; set; }
-    public IEnumerable<TestFile> TestFiles => TestProjects.SelectMany(testProject => testProject.TestFiles);
+
+    public IEnumerable<TestFile> TestFiles => TestProjects.SelectMany(testProject => testProject.TestFiles).Distinct();
```

Comment:
Is there no risk while setting the testcases on the test files that we might get one or the other instance of the same test file?